Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -167,7 +175,12 @@ func (s *simpleProvider) CreateContainerSecurityContext(pod *api.Pod, container
 		sc.RunAsNonRoot = &b
 	}
 
-	// No need to touch capabilities, they will validate or not.
+	caps, err := s.capabilitiesStrategy.Generate(pod, container)
```

Comment:
Are there going to be any other types of strategies that apply in lieu of the must run as strategy?